ngageoint/opensphere
# Layers Search Grid

NOTE: This needs the additional plugin to be testable as part of the original ticket.  This capability can still be used in plain Opensphere when the "supportsGrid" flag on the DrawControlsCtrl is set to 'true'; or if a future plugin/feature wants to reuse this grid code as well.

Testing steps:

1. Set "supportsGrid" to true (this is NOT how it's committed), or use provided Surge link
2. Lower the brightness or opacity of the "Street Map Tiles" layer 
3. Draw a box, circle, and/or polygon
4. Remember, this is NOT how it's committed; the grid will NOT be shown normally
5. Verify a grid snapped to 0.25 degree lat/lon coordinates is drawn around the shape (this can be changed to 1-degree or 0.1-degree grid boxes by changing the setting in settings.json)
6. Verify the grid is not drawn when the search area is large
